docker/default.yml

```diff
@@ -23,6 +23,8 @@ services:
       - ./docker//docker_entrypoint.sh
     expose:
       - "8001"
+    environment:
+      - DJANGO_SETTINGS_MODULE=intel_owl.settings
```

Comment:
If this variable is required only for CI, it should be set in the specific docker-compose file (`ci.override.yml`)...or, even better, you can just insert this variable in the specific env_file: `env_file_app_ci`

Reply:
agree with @mlodic